homeassistant/components/actronair_neo/config_flow.py

```diff
@@ -0,0 +1,175 @@
+"""Setup config flow for Actron Neo integration."""
+
+import logging
+
+from actron_neo_api import ActronNeoAPI, ActronNeoAPIError, ActronNeoAuthError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import callback
+
+from .const import DOMAIN, ERROR_API_ERROR, ERROR_INVALID_AUTH, ERROR_NO_SYSTEMS_FOUND
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ActronNeoConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Actron Air Neo."""
+
+    VERSION = 3
+
+    def __init__(self) -> None:
+        """Initialize the config flow."""
+        self.api = None
+        self.ac_systems = None
+
+    async def async_step_user(self, user_input=None) -> config_entries.ConfigFlowResult:
```

Comment:
```suggestion
    async def async_step_user(self, user_input=None) -> ConfigFlowResult:
```
Also please add typing to user_input

Reply:
Fixed